app/Resources/translations/messages.en_GB.xliff

```diff
@@ -502,18 +519,38 @@ An e-mail with your activation code has been sent to the e-mail address %email%.
       <trans-unit id="59ae5f416e832eeec457e8f73d11d196bcf9c6e5" resname="ss.second_factor.list.title">
         <source>ss.second_factor.list.title</source>
         <target>Token Overview</target>
-        <jms:reference-file line="4">/../src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/views/SecondFactor/list.html.twig</jms:reference-file>
+        <jms:reference-file line="3">/../src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/views/SecondFactor/list.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="64cd3b12ab8e092173e5e99069d38afdaf811b16" resname="ss.second_factor.remote_vet.text.are_you_sure">
         <source>ss.second_factor.remote_vet.text.are_you_sure</source>
         <target>Are you sure you want to confirm your identity through a third party?</target>
         <jms:reference-file line="9">/../src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/views/RemoteVetting/remoteVet.html.twig</jms:reference-file>
       </trans-unit>
+      <trans-unit id="789743faa71a92af3b0c7580952c3a47e62048ac" resname="ss.second_factor.remote_vet.text.does_it_match">
+        <source>ss.second_factor.remote_vet.text.does_it_match</source>
+        <target>Is the supplied information correct?</target>
```

Comment:
I suggest adding more context to this translation. Here you could explain what is expected from the user at this point. And explain that the data displayed in this form is from the RV IdP. You c/should mention what RV IdP was used. And how the user should use this form.

Reply:
Personally I would like to have to form validated first before taking this effort.

Reply:
See feedback above